app/Admin/Controllers/Management/AccessController.php

```diff
@@ -49,6 +50,31 @@ public function index(Content $content)
             'member' => User::get(['id', 'firstname', 'lastname']),
         ]);
 
+        // Define the filter for parents.
+        $grid->filter(function($filter) {
+            // Get all poles (all assos with children).
+            $poles = Asso::whereIn('id', function($query) {
+                return $query->select('parent_id')->from('assos');
+            })->pluck('name', 'id');
+
+            $filter->where(function ($query) {
+                $query->whereHas('asso', function ($query) {
+                    $query->where('parent_id', '=', $this->input);
+                });
+            }, 'Parent')->select($poles);
```

Comment:
We might want to name it "Pole" in order for administrator to understand better. What do you think  about it?

Reply:
I fixed it in my last commit, and changed to filter name to Pôle.